src/ci/init_repo.sh

```diff
@@ -53,7 +53,7 @@ modules=($modules)
 use_git=""
 urls="$(git config --file .gitmodules --get-regexp '\.url$' | cut -d' ' -f2)"
 urls=($urls)
-for i in ${!modules[@]}; do
+for i in "${!modules[@]}"; do
```

Comment:
`${!` expands to the number of modules, which will never have spaces. So I don't think this change is worth making.
```
$ modules="$(git config --file .gitmodules --get-regexp '\.path$' | cut -d' ' -f2)"
$ echo "$modules"
src/tools/rust-installer
src/doc/nomicon
src/tools/cargo
src/doc/reference
src/doc/book
src/tools/rls
src/tools/rustfmt
src/tools/miri
src/doc/rust-by-example
library/stdarch
src/doc/rustc-dev-guide
src/doc/edition-guide
src/llvm-project
src/doc/embedded-book
src/tools/rust-analyzer
library/backtrace
$ modules=($modules)
$ echo ${!modules[@]}
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15
```

Reply:
I agree with you. It's very few likely to include space in these file name.